src/material/input/input.md

```diff
@@ -5,17 +5,17 @@
 
 ### `<input>` and `<textarea>` attributes
 
-All of the attributes that can be used with normal `<input>` and `<textarea>` elements can be used
+All of the attributes that can be used with `<input>` and `<textarea>` elements can be used
 on elements inside `<mat-form-field>` as well. This includes Angular directives such as `ngModel`
 and `formControl`.
 
 The only limitation is that the `type` attribute can only be one of the values supported by
-`matInput`.
+`matNativeControl`.
 
 ### Supported `<input>` types
 
 The following [input types](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) can
-be used with `matInput`:
+be used with `matNativeControl`:
```

Comment:
Is it intentional that we only mention `matNativeControl` instead of `matInput`?

Reply:
Yeah, I always found it weird that the docs talk about matInput even for cases where it's used on a select element.